curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md

```diff
@@ -28,7 +28,7 @@ tests:
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(MyToDoList)); return mockedComponent.find('MyToDoList').length === 1; })());
   - text: The first child of <code>MyToDoList</code> should be a <code>textarea</code> element.
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(MyToDoList)); return mockedComponent.find('MyToDoList').children().childAt(0).type() === 'textarea'; })());
-  - text: The third child of <code>MyToDoList</code> should be a <code>button</code> element.
+  - text: The second child of <code>MyToDoList</code> should be a <code>button</code> element.
```

Comment:
```suggestion
  - text: The third child of <code>MyToDoList</code> should be a <code>button</code> element.
```
As I mentioned in https://github.com/freeCodeCamp/freeCodeCamp/issues/38170#issuecomment-582799717 the button is the third child, and the second child is a `<br/>` element.  If anything needs changing, it's the seed.